src/components/options/index.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useReducer, useEffect } from 'react'
+import Button from '@material-ui/core/Button'
+import AppBar from '@material-ui/core/AppBar'
+import Typography from '@material-ui/core/Typography'
+import { makeStyles } from '@material-ui/core/styles'
+import Toolbar from '@material-ui/core/Toolbar'
+import Link from '@material-ui/core/Link'
+import Box from '@material-ui/core/Box'
+
+import Input from '../input'
+import Form from '../form'
+import {
+    optionsReducer,
+    initialState,
+    LOAD_OPTIONS,
+    ADD_NEW_CHARACTER,
+    SAVE_CHARACTER,
+    REMOVE_CHARACTER,
+} from './options.reducer'
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        flexGrow: 1,
+        marginBottom: theme.spacing(3),
+    },
+    menuButton: {
+        marginRight: theme.spacing(2),
+    },
+    title: {
+        flexGrow: 1,
+    },
+}))
+
+const Options = () => {
+    const classes = useStyles()
+    const [state, dispatch] = useReducer(optionsReducer, initialState)
+    const [isAddingNewCharacter, setIsAddingNewCharacter] = useState(false)
```

Comment:
Where is this state used? And if it's not being used, should we remove it?

Reply:
Good catch, this needs to be removed. Gotta add some linting rules for unused variables...